source/Minable.cpp

```diff
@@ -180,7 +178,8 @@ void Minable::Place(double energy, double beltRadius)
 	// Start the object off with a random facing angle and spin rate.
 	angle = Angle::Random();
 	spin = Angle::Random(energy) - Angle::Random(energy);
-	SetFrameRate(Random::Real() * 4. * energy + 5.);
+	double frameRate = GetFrameRate() ? GetFrameRate() : Random::Real() * 4. * energy + 5.;
```

Comment:
Given that the default value framerate is not zero, this is not going to do what you want. I suggest checking the `DataNode` to see if a framerate value is provided in `Minable::Load` and set a `bool useRandomFramerate` field in `Minable` to `false` (the default should be `true`).